src/indexer/index_writer.rs

```diff
@@ -1522,11 +1532,18 @@ mod tests {
         // multivalue fast field tests
         for segment_reader in searcher.segment_readers().iter() {
             let ff_reader = segment_reader.fast_fields().u64s(multi_numbers).unwrap();
+            // let bool_ff_reader = segment_reader.fast_fields().bools(multi_bools).unwrap();
```

Comment:
This line was panicking on every test but only in some Inputs with a panic message:
```rust
'called `Result::unwrap()` on an `Err` value: SchemaError("Field(multi_bools) data was not found")'
```
I think `MultiValue` bools work fine and the issue is with the `MultiValuedFastFieldReader` since the other tests don't fail. I will remove the comments when this is fixed.